EZSource/Classes/TableViewDataSource/SectionSource.swift

```diff
@@ -65,4 +65,20 @@ class SectionSource {
         self.sections = sections.replacingOccurrences(with: section, where: { $0.id == section.id })
     }
     
+    func deleteEmptySections() -> [DeleteSectionInfo] {
```

Comment:
here there is no delete, maybe we have to rename it

Reply:
It still deletes the section in SectionSource and then the info used to delete them on the TV level

Reply:
great thank you